docs/deployment/feature_gates.md

```diff
@@ -42,6 +42,7 @@ The following tables are a summary of the feature gates that you can set on diff
 | ShootSARotation                              | `true`  | `Beta`  | `1.51` |        |
 | HAControlPlanes                              | `false` | `Alpha` | `1.49` |        |
 | DefaultSeccompProfile                        | `false` | `Alpha` | `1.54` |        |
+| CoreDNSQueryRewriting                        | `false` | `Alpha` | `1.54` |        |
```

Comment:
I am not quite sure that we will make it with this PR for 1.54 as tomorrow it is expected to have the release milestone created and a new release in the next days.
Maybe to be on the safe side we can target 1.55. WDYT?

```suggestion
| CoreDNSQueryRewriting                        | `false` | `Alpha` | `1.55` |        |
```

Reply:
Addressed with latest commit.